ui/DesignSystem/Button.svelte

```diff
@@ -77,6 +68,7 @@
     box-shadow: 0 0 0 0;
     border-color: var(--color-foreground-level-1);
     color: var(--color-foreground-level-3);
+    transform: scale(1);
```

Comment:
We also need to remove `background-color` from this selector (`.vanilla.disabled:hover`).